### PR TITLE
Added styling to scrollbar according to the theme of the site

### LINKED ISSUE #117: Style the scrollbar according to the theme of the site
The white/grey-ish scrollbar looks really out of place currently, make it fit in.

### PR DESCRIPTION
**Pull request by pranavgoel29**
## ✔️ Checklist (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

image
#117

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute.

I probably should have made a small mockup of how I wanted this to look. It looks great, nothings wrong but I had other ideas:
- The green colour should not be involved, Update it to use a darker variation of the greys.
- The shape or form of the previous scrollbar should stay the same, I have a terrible experience with tiny(small width) scrollbars that I don't think I want contribute to other people's frustrations. The border radius is not needed I think

**Basically**: just update the colours of the default scrollbar

**Comment by pranavgoel29** [comment]
Ok sure will do that. 👍

**Comment by pranavgoel29** [comment]
I tried dark grey color as you suggested however, I think it is harder to see than the green one.

image

image

**Comment by pranavgoel29** [comment]
image

**Comment by pranavgoel29** [comment]
> Thanks for taking the time to contribute.
> 
> I probably should have made a small mockup of how I wanted this to look. It looks great, nothings wrong but I had other ideas:
> 
> * The green colour should not be involved, Update it to use a darker variation of the greys.
> * The shape or form of the previous scrollbar should stay the same, I have a terrible experience with tiny(small width) scrollbars that I don't think I want contribute to other people's frustrations. The border radius is not needed I think
> 
> **Basically**: just update the colours of the default scrollbar


Changed the color to this.


image

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Yeah that's not better than the first one, can you just go back the the first one you did? The one in the Screenshot section of the first PR comment, apologies.

**Comment by Ngoakor12** [comment]
Also don't forget to link the related issue so it can automatically close when I merge the PR.

**Comment by pranavgoel29** [comment]
> Yeah that's not better than the first one, can you just go back the the first one you did? The one in the Screenshot section of the first PR comment, apologies.

ok sure, no worries

**Comment by pranavgoel29** [comment]
> Also don't forget to link the related issue so it can automatically close when I merge the PR.

Linked

**Review by Ngoakor12 — APPROVED** [review]
Much better  :ok_hand:  thank you for taking the time to contribute :raised_hands: 

A better spec-d out issue would have definitely reduced the back and forth.

**Comment by pranavgoel29** [comment]
> Much better  :ok_hand:  thank you for taking the time to contribute :raised_hands: 
> 
> A better spec-d out issue would have definitely reduced the back and forth.

Yes, if the issue had some specs about the design but no worries. 😊

Looking forward to contributing more.
Good luck with the project. 👍🏻
